include/pg_tmgr.h

```diff
@@ -0,0 +1,29 @@
+/*-------------------------------------------------------------------------
+ *
+ * include/pg_tmgr.h
```

Comment:
pg_shard stands for postgres sharding and is the name of the extension (like pg_stat_statements, pg_partman, etc.). This file provides transaction management function specific to pg_shard, not postgres in general. I guess pgs_tmgr.c would be more accurate, but we usually prefer words over acronyms.